app/src/main/cpp/openxr-context.h

```diff
@@ -1,5 +1,21 @@
 #pragma once
 
+#include "egl-instance.h"
+
+namespace zen::display_system::oculus {
+
 struct OpenXRContext {
   XrInstance instance{XR_NULL_HANDLE};
+  XrSystemId system_id{XR_NULL_SYSTEM_ID};
+  XrViewConfigurationType view_configuration_type{};
+  XrEnvironmentBlendMode environment_blend_mode{};
+
+  std::unique_ptr<EglInstance> egl;
+
+  static const XrViewConfigurationType kAcceptableViewConfigType =
+      XR_VIEW_CONFIGURATION_TYPE_PRIMARY_STEREO;
+  static const XrEnvironmentBlendMode kAcceptableEnvironmentBlendModeType =
+      XR_ENVIRONMENT_BLEND_MODE_OPAQUE;
```

Comment:
I think constant in C++ like these should be constexpr. What's your opinion?

Reply:
Yes, you are right!